zebra-consensus/src/checkpoint.rs

```diff
@@ -265,10 +265,11 @@ where
         let (sender, receiver) = mpsc::channel();
 
         #[cfg(feature = "progress-bar")]
-        let queued_blocks_bar = howudoin::new().label("Queued Checkpoint Blocks");
+        let queued_blocks_bar = howudoin::new_root().label("Checkpoint Queue Height");
 
         #[cfg(feature = "progress-bar")]
-        let verified_checkpoint_bar = howudoin::new().label("Verified Checkpoints");
+        let verified_checkpoint_bar =
```

Comment:
I ran into the `overflow when adding durations` panic again, I think it was this `verified_checkpoint_bar` because it's the only one calling `set_len()` that hadn't shown up when it panicked:

```console
The application panicked (crashed).
Message:  overflow when adding durations
Location: library/core/src/time.rs:920
```

<details>

```console
Metadata:
version: 1.0.0+91.gdfe2f85
Zcash network: Mainnet
state version: 25
features: default,getblocktemplate_rpcs,howudoin,indicatif,progress_bar,release_max_level_info
branch: fix-progress-bar-order
git commit: dfe2f85
commit timestamp: 2023-06-30T00:09:04.000000000Z
target triple: x86_64-unknown-linux-gnu
rust compiler: 1.70.0
rust release date: 2023-05-31
optimization level: 3
debug checks: false

</details>

Reply:
> The `verified_checkpoint_bar.set_len()` call wasn't added in this PR but may cause panics when starting past the last checkpoint.

This code isn't called when we start past the last checkpoint, or when we sync through the last checkpoint. The `BlockVerifierRouter` makes sure the `CheckpointVerifier` never runs on blocks past the last checkpoint.

This is actually a bug in the estimates and calculations in the `indicatif` crate. It will be fixed in their next release:
https://github.com/console-rs/indicatif/pull/555

There's nothing we can do about this bug in our code.  (We could remove the lengths, but in this case the length is actually relevant.)